site/docsource/Installation.adoc

```diff
@@ -1,45 +1,62 @@
 ## Installation
 
-Below is a list of different ways to install BuckleScript:
-
-### Windows Installation
+### From https://www.npmjs.com/package/bs-platform[npm] (recommended for normal users)
 
 [source,sh]
 ------------------------
-npm install bs-platform
+npm install -g bs-platform
 ------------------------
 
-### *nix installation
+It supports all platform including Windows
+
+### Build from source in Release Mode
+
+Note for dev mode, you are encouraged to work from https://github.com/BuckleScript/bucklescript/wiki/Contribute[there]
 
 .Prerequisites
-* Standard C toolchain
-* `npm` (should be installed with Node)
+* Standard C toolchain(gcc,Makefile)
 
-// * https://github.com/ninja-build/ninja/releases[ninja] (version >= 1.7)
+.Get source 
+
+[source,sh]
+-----------
+git clone https://github.com/bucklescript/bucklescript
+-----------
 
-The standard `npm` package management tool can be used to install
-BuckleScript. If you don't already have `npm` installed, follow the
-directions listed
-https://docs.npmjs.com/getting-started/installing-node[here]. Once `npm`
-is installed, run the following command:
+The patched compiler is installed locally into your `$(pwd)/bin`
```

Comment:
@bobzhang is this referring to `vendor/ocaml/bin` ?
If so, this section and the next could be swapped.